Wrappers/Python/cil/processors/CofR_image_sharpness.py

```diff
@@ -151,18 +151,18 @@ def check_input(self, data):
     def _configure_FBP(self, backend='tigre'):
         """
         Configures the processor for the right engine. Checks the data order.
-        """        
+        """
         if backend not in self._supported_backends:
             raise ValueError("Backend unsupported. Supported backends: {}".format(self._supported_backends))
 
         #set FBPOperator class from backend
         try:
-            module = importlib.import_module('cil.plugins.'+backend)
-        except ImportError:
+            module = importlib.import_module(f'cil.plugins.{backend}')
+        except ImportError as exc:
             if backend == 'tigre':
-                raise ImportError("Cannot import the {} plugin module. Please install TIGRE or select a different backend".format(self.backend))
+                raise ImportError(f"Please install `conda install tigre` or select a different backend") from exc
```

Comment:
```suggestion
                raise ImportError(f"Please install `conda install ccpi::tigre` or select a different backend") from exc
```

Reply:
although, I am not sure that is the actual message we want to show. I'd be more generic as earlier and say, please install TIGRE, or ASTRA (below) without saying use conda.